modules/foreman_search_facts.py

```diff
@@ -91,28 +90,14 @@
   type: list
 '''
 
-try:
-    from ansible.module_utils.ansible_nailgun_cement import (
-        search_entities_json,
-    )
-
-    from nailgun import entities
-    from nailgun.entity_mixins import EntitySearchMixin
-except ImportError:
-    pass
-
-from ansible.module_utils.foreman_helper import ForemanAnsibleModule
-
-
-def nailgun_entites():
-    return list(map(lambda entity: entity.__name__, EntitySearchMixin.__subclasses__()))
+from ansible.module_utils.foreman_helper import ForemanApypieAnsibleModule
 
 
 def main():
 
-    module = ForemanAnsibleModule(
+    module = ForemanApypieAnsibleModule(
```

Comment:
Is this name going to stick, once we ridded nailgun?

Reply:
As soon as we're back to one backend, we can switch the naming back. I don't mind either way.